src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs

```diff
@@ -32,6 +32,12 @@ public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
             }
         }
 
+        public LoggerPhaseScope(string phaseName, LogLevel perfLogLevel)
```

Comment:
>LoggerPhaseScope [](start = 15, length = 16)

how about the ctor with enablePerformanceScope to call the ctor with perfLogLevel?